lib/git_reflow/commands/start.rb

```diff
@@ -1,17 +1,27 @@
 desc 'Start will create a new feature branch and setup remote tracking'
 long_desc <<LONGTIME
   Performs the following:\n
-  \t$ git pull origin <current_branch>\n
-  \t$ git push origin <current_branch>:refs/heads/[new_feature_branch]\n
+  \t$ git checkout <base_branch>\n
+  \t$ git pull origin <base_branch>\n
+  \t$ git push origin <base_branch>:refs/heads/[new_feature_branch]\n
   \t$ git checkout --track -b [new_feature_branch] origin/[new_feature_branch]\n
 LONGTIME
-arg_name '[new-feature-branch-name] - name of the new feature branch'
+arg_name '[new-feature-branch-name] - name of the new feature branch, [base-branch-name] - name of the base branch you want to start off of'
```

Comment:
I agree that the order of parameters could be confusing. Perhaps it is better to have the flag afterall.